test/integration/user_cannot_view_another_users_data_test.rb

```diff
@@ -12,7 +12,7 @@ class AuthenticatedUserPrivacyTest < ActionDispatch::IntegrationTest
     visit '/users'
     assert page.has_content?("Oops!")
 
-    visit '/user/1'
+    visit '/users/1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.